CONTRIBUTING.md

```diff
@@ -8,7 +8,10 @@ The team gratefully accepts contributions via [pull requests](https://help.githu
 
 The project has integration tests based on `scripted test framework`.
 See https://www.scala-sbt.org/1.x/docs/Testing-sbt-plugins.html
-Command to run the tests: `sbt ';sbt-bom / scripted'`
+Test splitted into 2 groups 
+- `psv` - run on any branch. These tests group **mustn't** rely on the secrets
+- `sv` - run only on master.
+Command to run the psv tests: `sbt ';sbt-bom / scripted psv/*'`
```

Comment:
I would add two sections before on how to:

# Build

# Run Tests

# Continuous Integration

Reply:
Done